docs/reference/model.rst

```diff
@@ -1,53 +1,138 @@
-.. automodule:: stravalib.model
-    :inherited-members: BaseModel
-
-.. If you use both automodule and autosummary you duplicate autodocs efforts
-.. Summary Functions
-.. -------------------------------
-.. .. autosummary::
-..    :toctree: api/
-
-..    lazy_property
-..    check_valid_location
-..    naive_datetime
-
-.. Summary Classes
-.. -------------------------------
-.. .. autosummary::
-..    :toctree: api/
-
-..    DeprecatedSerializableMixin
-..    BackwardCompatibilityMixin
-..    BoundClientEntity
-..    LatLon
-..    Club
-..    Gear
-..    Bike
-..    Shoe
-..    ActivityTotals
-..    AthleteStats
-..    Athlete
-..    ActivityComment
-..    ActivityPhotoPrimary
-..    ActivityPhotoMeta
-..    ActivityPhoto
-..    ActivityKudos
-..    ActivityLap
-..    Map
-..    Split
-..    SegmentExplorerResult
-..    AthleteSegmentStats
-..    AthletePrEffort
-..    Segment
-..    SegmentEffortAchievement
-..    BaseEffort
-..    BestEffort
-..    SegmentEffort
-..    Activity
-..    DistributionBucket
-..    BaseActivityZone
-..    Stream
-..    Route
-..    Subscription
-..    SubscriptionCallback
-..    SubscriptionUpdate
+============
+Model
+============
+
+.. currentmodule:: stravalib.model
```

Comment:
I understand that this approach means more maintenance. But the docs are infinently more readable if we organize them this way. i'll go back and double check each class / method here (i did this but not for things we've removed - just for new classes. 

so i propose we try this out and if it does end up being too hard to maintain i'll revert to automodule.